YoLetsChillFrontend/src/components/pages/SplashPage/SplashPage.module.css

```diff
@@ -0,0 +1,67 @@
+.container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 2rem;
+    min-height: 100vh;
+    background-color: #e6f0ea; /* soft forest greenish background */
+    color: #1e3d32; /* deep forest green */
+    font-family: 'Inter', sans-serif;
+    text-align: center;
+  }
+  
+  .title {
+    font-size: 2.5rem;
+    font-weight: 700;
+    margin-bottom: 1rem;
+  }
+  
+  .description {
+    font-size: 1.2rem;
+    max-width: 90%;
+    margin-bottom: 2rem;
+    line-height: 1.6;
+  }
+  
+  .ctaButton {
+    background-color: #2e5e4f;
+    color: #ffffff;
+    font-size: 1.2rem;
+    padding: 1rem 2rem;
+    border: none;
+    border-radius: 12px;
+    cursor: pointer;
+    margin-bottom: 2rem;
+    transition: background-color 0.3s ease;
+  }
+  
+  .ctaButton:hover {
+    background-color: #244d40;
+  }
+  
+  .infoSection {
+    font-size: 1rem;
+    line-height: 1.8;
+    max-width: 80%;
+  }
+  
+  @media (min-width: 768px) {
+    .title {
+      font-size: 3.5rem;
+    }
+  
+    .description {
+      font-size: 1.5rem;
+    }
+  
+    .ctaButton {
+      font-size: 1.4rem;
+      padding: 1.2rem 2.5rem;
+    }
+  
+    .infoSection {
+      font-size: 1.2rem;
+    }
+  }
+  
```

Comment:
Not sure why there's a no-newline complaint here. Maybe 67 has some unexpected whitespace?